extension/src/popup/views/Account/index.tsx

```diff
@@ -158,13 +134,12 @@ export const Account = () => {
             </div>
           </div>
         </div>
-        <div>
+        <div className="AccountView__assets-wrapper">
```

Comment:
changed the AccountAssets "wrapper" to the component using it so can control the row spacing there